src/components/game/components/cards.js

```diff
@@ -0,0 +1,124 @@
+import React, {useState, useContext, useEffect} from 'react';
+import ItemCard from "./item-card";
+import {Context, Context2, Context4} from "../../context";
+import {connect} from "react-redux";
+import getCards from "../../../utils/getCards";
+
+const SIZE = {
+	'2*4': 8,
+	'3*4': 12,
+	'3*6': 18,
+};

```

Comment:
you use this const in many files, it is better to save such constant in separate constants file and import it

Reply:
I agree with you )) I was just too lazy to do so right away